docs/network/smart-rollup-nodes.md

```diff
@@ -18,65 +18,92 @@ Make sure that you understand the interaction between different nodes as describ
+- [Running the Smart Rollup node in operator mode](#running-the-smart-rollup-node-in-operator-mode)
+- [Running the Smart Rollup node in observer mode](#running-the-smart-rollup-node-in-observer-mode)
 
-For simplicity, these steps show how to run the Smart Rollup node in observer mode:
+In either case, you can start the Smart Rollup node with a snapshot of the Etherlink state to prevent it from having to compute the state from Etherlink genesis.
+You can also skip the snapshot and run the node from Etherlink genesis, but it can take a long time for the node to catch up.
+These instructions cover both ways of running the Smart Rollup node.
+
+## Running the Smart Rollup node in operator mode
+
+Running the Smart Rollup node in operator mode is the best way to participate in Etherlink because in this mode the node posts commitments about Etherlink's state to layer 1.
+However, running the node in this mode requires an account with at least 10,000 staked tez and certain other prerequisites.
+
+To run an Etherlink Smart Rollup node in operator mode, you need:
+
+- A Tezos layer 1 node running in archive mode or full mode with expanded node history, such as with the argument `--history-mode full:50`
+- An account with at least 10,000 tez staked, referred to as the _operator account_
+- A separate account to sign batching operations, referred to as the _batcher account_; this account is necessary only if you also run an Etherlink EVM node
+- A clean data directory that has not been used for another Smart Rollup node or a node running in a different mode
 
 1. Get a built version of the Smart Rollup node binary, named `octez-smart-rollup-node`.
 The best place to get the most recent binary files to use with Etherlink is https://gitlab.com/tezos/tezos/-/releases.
 
 1. Initialize the local context of the node, which is where it stores local data:
 
-   1. Set the environment variable `sr_observer_data_dir` to the directory where the node should store its local data.
+   1. Set the environment variable `SR_DATA_DIR` to the directory where the node should store its local data.
    The default value is `$HOME/.tezos-smart-rollup-node`.
-   1. Initialize the local context by running this command:
+
+   1. Initialize the local context by running this command and using the address of the Etherlink Smart Rollup, which you can get from the [Network information](/get-started/network-information) page, and the addresses or Octez client aliases for the operator account and the batcher account:
 
       ```bash
-      octez-smart-rollup-node init observer config for sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf \
-        with operators --data-dir $sr_observer_data_dir \
+      octez-smart-rollup-node init operator config for sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf \
+        with operators \
+        operating:$OPERATOR_ACCOUNT \
+        cementing:$OPERATOR_ACCOUNT \
+        batching:$BATCHER_ACCOUNT \
+        executing_outbox:$OPERATOR_ACCOUNT \
```

Comment:
```suggestion
        $OPERATOR_ACCOUNT \
```

Need to check but IIRC this works.

Reply:
Current version seems necessary for Octez v20. Can’t wait for V21 to be the stable release.